src/dex-ui/pages/Withdraw/Withdraw.tsx

```diff
@@ -84,10 +84,14 @@ const Withdraw = () => {
       }
     } else {
       // no pool indicated, so redirect to My Pools page
-      navigate("/pool?selectedPools=user");
+      const navigateToUserPools = async () => {
+        await pools.navigateToPoolsPage(1);
```

Comment:
Note for the future - we should move this to a central store navigation function (i.e. `app.navigateToPage`).